libfwupdplugin/fu-device.h

```diff
@@ -504,9 +507,9 @@ fu_device_get_firmware_size_min(FuDevice *self);
 guint64
 fu_device_get_firmware_size_max(FuDevice *self);
 guint
-fu_device_get_progress(FuDevice *self);
+fu_device_get_progress(FuDevice *self) G_DEPRECATED_FOR(fu_progress_get_percentage);
 void
-fu_device_set_progress(FuDevice *self, guint progress);
+fu_device_set_progress(FuDevice *self, guint progress) G_DEPRECATED_FOR(fu_progress_set_percentage);
```

Comment:
didn't we break ABI already in 1.7.0 but not release yet?  Why not just axe the `fu_device_get_progress` and `fu_device_set_progress` _now_?

Reply:
It was more to avoid upsetting the people working on new plugins (some public, some still secret) that are being worked on behind the scenes. Perhaps we can rip out the symbols before we do the 1.7.0 release?

Reply:
I ripped them out.